src/shared/components/FilterList/InjectionOption.tsx

```diff
@@ -30,25 +31,39 @@ interface Props {
   setHoverdFunction?: (string: string) => void
 }
 
-const defaultProps = {
-  testID: 'flux-injection-option',
-}
-
 const FluxInjectionOption: FC<Props> = ({
   option,
   extractor,
   onClick,
-  testID,
+  testID = 'flux-injection-option',
   ToolTipContent,
   searchTerm,
   setToolTipPopup,
   setHoverdFunction,
 }) => {
+  // TODO: move this to a flag once design wants to branch it out further
+  const {fluxQueryBuilder} = useContext(AppSettingContext)
```

Comment:
this triggers the styling changes that are only applied to the function browser in the new data explorer

Reply:
The new view only shows up on the new data explorer and not in the old data explorer. Design is unclear about the roll out plan, so it's not just a flag that is rolled out everywhere

Reply:
Using a globally set value `fluxQueryBuilder`, which toggles on in the new QX FluxBuilder, means that everywhere else in the codebase needs to reset this global value back to false --> whenever we navigate away from the new QX FluxBuilder.

Otherwise this new style change starts popping up in other parts of the application:

https://user-images.githubusercontent.com/10232835/174398386-a3f10288-0cb4-48e6-8fb2-7c0331735505.mov